### DISCUSSION TITLE
it is blank in localhost3000 when I run yarn broswer start successfully

### DISCUSSION THREAD

**Discussion by wanshiyuan** [eclipse-theia/theia#13859] [Q&A]
I have done following steps
 git clone https://github.com/eclipse-theia/theia.git \
    && cd theia \
    && yarn \
    && yarn browser build \
    && yarn browser start
but it is blank , I thought  an ide should be displayed in the page
![image](https://github.com/eclipse-theia/theia/assets/64078754/9e061ba5-5147-43b8-8c25-584bcd2121ba)

**Comment by msujew** [comment]
Hey @wanshiyuan,

Do you get any networl errors when loading the page? I'm a bit concerned about the hash on the js bundle - it's not supposed to create that hash and the file is usually only available as `bundle.js` on the backend.

**Comment by icoco** [comment]
```

        yarn clean 

        yarn --pure-lockfile && \
        yarn build:extensions && \
        yarn download:plugins && \ // ignore this step no plugin support 
        yarn browser build

        yarn browser start 
```

try above
